cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -143,4 +123,30 @@ private void notifyPlugins(Project project, Target target) {
             plugin.codeGenerated(project, target);
         }
     }
+
+    private Path getExecutablePath(Project project) {
+
+        Path fileName = project.sourceRoot().getFileName();
+
+        // If the --output flag is not create the executable in the current directory
```

Comment:
// If the --output flag is not **set,** ...

Reply:
Fixed